features/rum/src/appleMain/kotlin/com/datadog/kmp/rum/configuration/RumConfiguration.kt

```diff
@@ -28,6 +33,77 @@ fun RumConfiguration.Builder.trackUiKitViews(
     return this
 }
 
+/**
+ * Enable automatic tracking of SwiftUI views as RUM views.
+ *
+ * RUM will query this predicate for each SwiftUI view detected through hosting controllers. The SDK extracts
+ * view names from the SwiftUI view hierarchy within those controllers, then passes those names to this predicate to
+ * determine which views should be tracked. The predicate implementation should return RUM view parameters if the
+ * given view should be tracked, or `null` to ignore it.
+ *
+ * [DefaultSwiftUIRUMViewsPredicate] will be used by default, or you can create your own predicate
+ * by implementing [SwiftUIRUMViewsPredicate].
+ *
+ * This API is experimental and may change in future. If the result is not good enough, please use manual
```

Comment:
Perhaps this is an opportunity to ask the customer for feedback if they're not seeing good results.